docs/sphinx/examples/cpp/other/trotter_kernel_mode.cpp

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// Compile and run with:
+// ```
+// nvq++ -std=c++17 --enable-mlir -v trotter_kernel_mode.cpp -o temp && ./temp
```

Comment:
Is `-std=c++17` compulsory? Otherwise, I'd think it's better to use the default (not specifying the standard).

Reply:
Also, I'd recommend using a more descriptive output, like `trotter.x`.

Reply:
Not compulsory. just copied from out tests in github:) Will do!